app/templates/suppliers/details.html

```diff
@@ -169,4 +169,17 @@
   {% endif %}
 
 {% endcall %}
+
+{% if currently_applying_to %}
+  You must complete your company details to make an application.
```

Comment:
Does this line need to be here? I see that it's in the screenshot from the prototype but I'm not sure it's part of this ticket? It feels weird to conditionally display it depending on the framework status. Worth checking with @laurenceberry?

Other than that, this looks good to me!

Reply:
This is fine for now but it's a sentence that will change once we build the green confirm button.